config/locales/en.yml

```diff
@@ -111,179 +111,139 @@ en:
       start_now: Start now
       what_you_need_to_know: What you need to know
   roadmap:
+    page_title: Roadmap - GOV.UK
+    hero:
+      description: GOV.UK is the trusted source of government information, so we’re always working to better meet user needs. This is our current roadmap, which takes us through to late 2021.
+      heading: GOV.UK roadmap
+    updates_section:
+      heading: Programme updates
+      items:
+      - href: "https://gds.blog.gov.uk/2021/03/29/whats-next-for-gov-uk-in-2021-to-2022/"
+        image_src: "roadmap/updates-goals.jpg"
+        image_alt: "alt text"
```

Comment:
These images are decorative. Can we please leave alt text blank so that screen readers skip over them?